examples/js/loaders/GLTFLoader.js

```diff
@@ -1241,9 +1242,10 @@ THREE.GLTFLoader = ( function () {
 			var scenes = dependencies.scenes || [];
 			var scene = scenes[ json.scene || 0 ];
 			var animations = dependencies.animations || [];
+			var asset = json.asset;
 			var cameras = dependencies.cameras || [];
 
-			onLoad( scene, scenes, cameras, animations );
+			onLoad( scene, scenes, cameras, animations, asset );
```

Comment:
What about passing `json` directly?

Reply:
would be possible of course, but then we would pass the whole GLTF data back.
In my opinion this is a bit of overkill so I would stick with the ```asset```. Or are there any other values in the GLTF which would be good to know as caller of the loader? I am not so familiar with the structure of the GLTF format